proposals/zips/zip001.sol

```diff
@@ -144,21 +156,11 @@ contract zip001 is Proposal {
         // Sum of Role counts to date
         assertEq(_core.getRoleMemberCount(Roles.LOCKER_PROTOCOL_ROLE), 2, "incorrect locker count");
         assertEq(_core.getRoleMemberCount(Roles.MINTER_PROTOCOL_ROLE), 2, "incorrect minter count");
-    }
 
-    function _validateOnChain(Addresses, address deployer) internal override {
-        /// Verify ADMIN role has been revoked from deployer
-        assertEq(_core.hasRole(Roles.ADMIN, deployer), false, "deployer should not have admin role");
+        console.log("Verify ADMIN role has been revoked from deployer on mainnet");
+        // assertEq(_core.hasRole(Roles.ADMIN, deployer), false, "deployer should not have admin role");
 
-        /// Verify only ADMIN_MULTISIG has ADMIN role
-        assertEq(_core.getRoleMemberCount(Roles.ADMIN), 1, "incorrect admin count");
+        console.log("Verify only ADMIN_MULTISIG has ADMIN role on mainnet");
+        // assertEq(_core.getRoleMemberCount(Roles.ADMIN), 1, "incorrect admin count");
```

Comment:
don't we need this anymore?

Reply:
These checks were only applicable on mainnet as on mainnet deployer admin role was being revoked. To remove `_validateOnChain`, I have commented these checks. I have not deleted the code, have commented it so that in case in future if in a very rare scenario we want a testnet deployment exactly as mainnet we don't forget these checks.

Reply:
The first check is specific to the mainnet. For eg: 
`assertEq(_core.hasRole(Roles.ADMIN, deployer), false, "deployer should not have admin role");`
This check is not applicable for testnets as on test networks we don't revoke Admin role from deployer.

The second check can be updated for testnet. Will add that.